torch/distributed/utils.py

```diff
@@ -0,0 +1,65 @@
+import torch
+
+def _parse_remote_device(remote_device: str):
+    r"""
+    Parses the remote device.
+
+    Args:
+        remote_device (str): Device on the destination worker where we'd like to place this module.
+            The format should be one of the following:
+
+                1. "<workername>/<device>", where the device field can be parsed as torch.device type.
+                   E.g., "trainer0/cpu", "trainer0", "ps0/cuda:0".
+                   In addition, the device field can be optional and the default value is "cpu".
+                2. "rank:<rank>/<device>", where <rank> is the rank of the
```

Comment:
Thanks for unifying the util functions for both sharded tensor and remote module! This means that wow we are also supporting the second format for `RemoteModule`. Is there any documentation that we should update for `RemoteModule`?

Reply:
It may be even better to just reuse `DevicePlacement` class in `RemoteModule`, then we don't need to maintain two copies of valid formats.